RetailSuiteAccounting/Integration/03_ManageAccounts.md

```diff
@@ -0,0 +1,131 @@
+[!!User Interface accounts](../UserInterface/02b_Accounts.md)  
+[!!Review an account](../Operation/02_ReviewAccount.md)  
+
+# Manage the accounts
+
+The accounts are set up during the configuration process via the accounting wizard, see [Charts of accounts](./01_RunAccountingWizard.md#chart-of-accounts). However, you can create, edit or delete an account if necessary in the *Accounting* module, for example if a specific account is not yet available in the system or the settings of an existing one need to be adjusted.
+
+> [Warning] Any modifications in the accounts have far-reaching consequences for the *Accounting* module and the system automatic bookings. It is highly recommended not to make any changes in the accounts, unless you have an in-depth knowledge of their functions. In case of doubt, contact our support team.
```

Comment:
```suggestion
> [Info] Any modifications in the accounts have far-reaching consequences for the *Accounting* module and the system automatic bookings. It is highly recommended not to make any changes in the accounts, unless you have an in-depth knowledge of their functions. In case of doubt, please contact the Customer Support.
```